apps/backend/src/donations/dto/filter-donations.dto.ts

```diff
@@ -1,17 +1,27 @@
-import { IsDate, IsDateString, IsEnum, IsInt } from 'class-validator';
+import {
+  IsDate,
+  IsDateString,
+  IsEnum,
+  IsInt,
+  IsOptional,
+} from 'class-validator';
 import { DonationStatus } from '../types';
 import { Timestamp } from 'typeorm';
 
 export class FilterDonationsDto {
   @IsDateString()
```

Comment:
Do @IsDate() instead to match with the Date types in DTO

Reply:
I tried this and it broke. Is there anything wrong with isDateString?